docs/reference/function/with_transaction.txt

```diff
@@ -21,11 +21,27 @@ Definition
 
    .. code-block:: php
 
-      function with_transaction(MongoDB\Driver\Session $session, callable $callback, array $transactionOptions = []): void
+      function with_transaction(
+          MongoDB\Driver\Session $session,
+          callable $callback,
+          array $transactionOptions = []
```

Comment:
Just checking to confirm that the missing trailing comma is intentional. This functionality was only [added in PHP 8.0](https://php.watch/versions/8.0/trailing-comma-parameter-use-list), so if we want to keep our docs PHP version consistent with the minimum library version, it would make sense to omit it.

Reply:
This was intentional in that I didn't think about it at all. I was only adding line breaks to the original content, which was on a single line.